### PR TITLE
WIP: Add skeleton of kansas city file

### PR DESCRIPTION
**Pull request by githubteacher**
WIP 🚧  

This file adds information and points of interest for KC. 

To do: 
- [ ] Add restaurants
- [ ] Add things to do 

Are there any types of food you'd like to try?

### CODE REVIEW

**Comment by mock** [comment]
fine. comment.

File: brianamarie-kansascity.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Kansas City
+
+## Places to Eat
+- Joe's KC BBQ (z man sandwich or burnt ends)
+- Jack Stack BBQ
```
Comment (githubteacher) [inline brianamarie-kansascity.md:5]: What should I get here?

File: brianamarie-kansascity.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+## Things to Do
+- Sporting Kansas City
+- Kansas City Royals
```
Comment (githubteacher) [inline brianamarie-kansascity.md:9]: Go Royals! :baseball:

File: brianamarie-kansascity.md (hunk 1 above)
Comment (githubteacher) [reply brianamarie-kansascity.md:5]: More comments

Comment (githubteacher) [reply brianamarie-kansascity.md:5]: Commment

**Comment by githubteacher** [comment]
:shipit:

**Pull request merged**